Code/Engine/Foundation/Math/Math.h

```diff
@@ -152,8 +152,8 @@ namespace ezMath
   constexpr ezInt32 FloatToInt(float value);
 
   // There is a compiler bug in VS 2019 targeting 32-bit that causes an internal compiler error when casting double to long long.
-  // FloatToInt(double) is not available on these version of the MSVC compiler.
-#if EZ_DISABLED(EZ_PLATFORM_ARCH_X86) || (_MSC_VER <= 1916)
+  // FloatToInt(double) is not available on these version of the msvc compiler.
```

Comment:
Why would you remove parenthesis and change the capitalization of MSVC ?

Reply:
I had a merge conflict there. Change was not intentional.